src/simd_basic_fnt.h

```diff
@@ -198,135 +198,6 @@ inline void add_props(
     }
 }
 
-/* ==================== Operations for RingModN =================== */
```

Comment:
This commit could be merged with ` [New] simd_ring.h includes accelerated operations dedicated for RingModN ` no?
You are onyl moving code, not modifying it right?

If yes, it's clearer to do the move in a single commit.

Reply:
yes, okay. I group commits concerning splitting `simd_basic` to `simd_fnt` and `simd_ring` into one commit.